src/System.Memory/src/System/Buffers/SequenceReaderExtensions_binary.cs

```diff
@@ -0,0 +1,173 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Re `SequenceReaderExtensions_binary.cs` .. our naming pattern seems more like `SequenceReaderExtensions.Binary.cs` eg `Process.Win32.cs`, `String.Searching.cs` etc.

Reply:
As I mentioned elsewhere, I was trying to match the existing file naming in this project. Since you're the second to bring it up I'll change it. It might be worth making a pass to normalize our file names a bit.